### DISCUSSION TITLE
Can't edit remote files

### DISCUSSION THREAD

**Discussion by tsvallender** [kovidgoyal/kitty#7211] [Q&A]
I want to be able to edit files locally from a current SSH section as the documentation suggests is possible, but I can't get it working.

I can connect via `kitty ssh session-name`, and get the menu of options by running `ls --hyperlink=auto`. If I choose to open the file, it opens in gedit (though `$EDITOR` is set to `nvim`), choosing to edit seems to do nothing at all.

Suggestions on fixing this appreciated, this would be a great feature to have working.

**Comment by kovidgoyal** [comment]
You want

kitten ssh hostname

not

kitty ssh hostname

and then simply run

edit-in-kitty filename

or do the ls and click routine if you prefer that.

**Reply by tsvallender** [reply]
This has the same behaviour for me, though I get command not found for `edit-in-kitty`. To be clear, you’re saying execute that as a normal command on the server you’ve connected to? I presume kitty is acting as a man in the middle or something similar?

**Comment by kovidgoyal** [comment]
ssh into the server using the ssh kitten, read more at:
https://sw.kovidgoyal.net/kitty/kittens/ssh/

**Comment by rrpolanco** [comment]
I ran into a similar issue. Ensure you are _not_ overriding the [editor envar](https://sw.kovidgoyal.net/kitty/glossary/#envvar-EDITOR) in your config.
